build_runner/README.md

```diff
@@ -99,14 +100,71 @@ directories are served, and on what ports.
 For example to serve the `example` and `web` directories on ports 8000 and 8001
 you would do `dart run build_runner serve example:8000 web:8001`.
 
-##### test
+### test
```

Comment:
```suggestion
##### test
```

This is intended to describe the command line options for `test`, so its a sub-heading of that